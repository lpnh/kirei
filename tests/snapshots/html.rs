#[test]
fn html_comment() {
    format_and_snapshot!("html/comment.html");
}

#[test]
fn doctype() {
    format_and_snapshot!("html/doctype.html");
}

#[test]
fn paragraph() {
    format_and_snapshot!("html/paragraph.html");
}

#[test]
fn link() {
    format_and_snapshot!("html/link.html");
}

#[test]
fn image() {
    format_and_snapshot!("html/image.html");
}

#[test]
fn list() {
    format_and_snapshot!("html/list.html");
}

#[test]
fn form() {
    format_and_snapshot!("html/list.html");
}

#[test]
fn table() {
    format_and_snapshot!("html/table.html");
}

#[test]
fn script() {
    format_and_snapshot!("html/script.html");
}

#[test]
fn style() {
    format_and_snapshot!("html/style.html");
}

#[test]
fn void_one() {
    format_and_snapshot!("html/void_one.html");
}

#[test]
fn void_two() {
    format_and_snapshot!("html/void_two.html");
}

#[test]
fn erroneous_end_tag() {
    format_and_snapshot!("html/erroneous_end_tag.html");
}

#[test]
fn no_text() {
    format_and_snapshot!("html/no_text.html");
}

#[test]
fn page() {
    format_and_snapshot!("html/page.html");
}
#[test]
fn scoops_and_dreams() {
    format_and_snapshot!("html/scoops_and_dreams.html");
}
