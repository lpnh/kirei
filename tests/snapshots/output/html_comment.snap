---
source: tests/snapshots/html.rs
description: "<!-- This is a comment -->\n"
---
<!-- This is a comment -->
